src/console/MeasurementCollectionToFhir/ProcessorStartup.cs

```diff
@@ -38,8 +38,8 @@ public void ConfigureServices(IServiceCollection services)
             services.Configure<ResourceIdentityOptions>(Configuration.GetSection("ResourceIdentity"));
             services.Configure<FhirClientFactoryOptions>(Configuration.GetSection("FhirClient"));
 
-            services.TryAddSingleton<IFactory<IFhirClient>, FhirClientFactory>();
-            services.TryAddSingleton(sp => sp.GetRequiredService<IFactory<IFhirClient>>().Create());
+            services.TryAddSingleton<IFactory<FhirClient>, FhirClientFactory>();
```

Comment:
Question, why we no longer using the IFhirClient but with FhirClient?

Reply:
@ms-teli it was removed in the latest version of the FHIR library.  Which is unfortunately since it complicates the testing.